config/test.exs

```diff
@@ -16,3 +16,6 @@ config :target, TargetWeb.Endpoint,
 
 # Print only warnings and errors during test
 config :logger, level: :warn
+
+# Config mock for mails
+config :target, :confirmation_mailer, TargetWeb.Mailer.InMemory
```

Comment:
Just checked, we can also use the [Bamboo.TestAdapter](https://github.com/thoughtbot/bamboo#using-adapters) 😁

Reply:
I don't know about this, since we don't use Bamboo Mailer directly, we have our own (because of Pow).
And delivery differs so I don't think I can use it, at least not directly.